src/ray/protobuf/serve.proto

```diff
@@ -208,3 +208,17 @@ message StatusOverview {
   ApplicationStatusInfo app_status = 1;
   DeploymentStatusInfoList deployment_statuses = 2;
 }
+
+// RPC Schema
+message PredictRequest {
+  string model_name = 1;
+  map<string, bytes> input = 2;
```

Comment:
If we require the client to understand and deserialize the input anyway, why not just have `bytes` instead of `map<string, bytes>`?  Or is this standard (like is this how they do it in torchserve?)

Reply:
Yeah, that is normal use case.  Dict{feature_name: data}